docs/understanding_reports.md

```diff
@@ -0,0 +1,38 @@
+# Understanding statistics
+
+This document is here to help you understand reports from [the nightly publishing job](https://github.com/open-vsx/publish-extensions/actions/workflows/publish-extensions.yml).
+
+## How to gather these statistics
+
+If you click on a job in the GitHub Actions tab, there is an `Artifacts` section at the bottom of the page, from which you can download the `report`, which after unarchiving reveals two files: `result.log` and `stat.json`.
+
+## `stat.json`
+
+This is the machine-readable data that the next file - `result.log` is generated from. In it, you can find 9 different categories of extensions:
+- `upToDate` - these extensions are the extensions, which have the same version published to OpenVSX as well as the Microsoft Marketplace.
+- `outdated` are all of the extensions, which have versions on OpenVSX, which are behind the ones on the Microsoft Marketplace. 
+- `unstable` is the category, which has all extensions, that are most likely incorrectly published from the extensions' nightly or beta builds, since on the Microsoft Marketplace has a version which is smaller than the one on OpenVSX.
+- `notInOpen` includes extensions that simply failed to ever be published to OpenVSX, which means they should get special attention - fix them or remove them :)
+- `notInMs` - extensions that aren't published on the Microsoft Marketplace
+- `failed` - the extensions that for some reason failed with their publishing.
+- `msPublished` - all extensions published by Microsoft Corporation.
+- `hitMiss`
```

Comment:
Are we missing a description? 😁

Reply:
Yes we are, I need to figure out what this thing does 😅

Reply:
Fixed in https://github.com/open-vsx/publish-extensions/pull/556/commits/78632d51ba4b9fda2e1ab4b97d6ffaddeddfe39b :)